layer_platform/PlatformBar.qml

```diff
@@ -134,7 +134,7 @@ ListView {
                 height: width
                 smooth: true
                 fillMode: Image.PreserveAspectCrop
-                source: gameData ? gameData.assets.screenshots[0] || "" : ""
+                source: gameData.assets.background
```

Comment:
Same deal as the comment on the SoftwareScreen - I get what you're going for, but as someone who doesn't have fanart set for all my games, but has screenshots, I prefer screenshots. Maybe I can do an option in the future to toggle which to display, backgrounds instead screenshots